.github/workflows/odbc-tests.yml

```diff
@@ -33,22 +33,57 @@ jobs:
         if: always() && steps.build-extensions.outcome == 'success'
         uses: ./.github/composite-actions/install-extensions
 
-      - name: Install ODBC Driver
-        id: install-odbc-driver
+      - name: Install SQL Server ODBC Driver
+        id: install-sql-server-odbc-driver
         if: always() && steps.install-extensions.outcome == 'success'
         run: |
           cd ~
           sudo apt-get install msodbcsql17
       
+      - name: Install unixODBC Driver
+        id: install-unix-odbc-driver
+        if: always() && steps.install-extensions.outcome == 'success'
+        run: |
+          cd ~
+          wget http://www.unixodbc.org/unixODBC-2.3.11.tar.gz
+          gunzip unixODBC*.tar.gz
+          tar xvf unixODBC*.tar
+          cd unixODBC-2.3.11
+          ./configure
+          make
+          sudo make install
+
+      - name: Install psqlODBC Driver
+        id: install-psql-odbc-driver
+        if: always() && steps.install-extensions.outcome == 'success' && steps.install-unix-odbc-driver.outcome=='success'
+        run: |
+          cd ~
+          wget https://ftp.postgresql.org/pub/odbc/versions/src/psqlodbc-12.01.0000.tar.gz
+          tar -zxvf psqlodbc-12.01.0000.tar.gz
+          cd psqlodbc-12.01.0000
+          ./configure
+          sudo make
+          sudo make install
+          echo '[ODBC_Driver_12_for_PostgreSQL]' | sudo tee -a /etc/odbcinst.ini > /dev/null
+          echo 'Description=ODBC Driver 12 for PostgreSQL Server' | sudo tee -a /etc/odbcinst.ini > /dev/null
+          echo 'Driver=/usr/local/lib/psqlodbcw.so' | sudo tee -a /etc/odbcinst.ini > /dev/null
+          echo 'UsageCount=1' | sudo tee -a /etc/odbcinst.ini > /dev/null
+      
       - name: Run ODBC Tests
-        if: always() && steps.install-odbc-driver.outcome == 'success'
+        if: always() && steps.install-sql-server-odbc-driver.outcome == 'success' && steps.install-psql-odbc-driver.outcome == 'success'
```

Comment:
Most likely won't need the `always()`
And ditto to all the other places which has `always()`

Reply:
Done. Removed the `always()` on steps pertaining to ODBC. Left the `always()` for the steps that involved installing & building the BBF server because many other yml files seem to do this, and there must be some particular reason why it's done this way.